src/common/ui-components/Map/PinIcon/PinIcon.module.scss

```diff
@@ -0,0 +1,3 @@
+.offset {
```

Comment:
any specific reason for this offset?

Reply:
It's to align the point of the pin with the coordinates of the feature.

Reply:
why exactly this much?

Reply:
Can I see the image?

Reply:
Its height and width / 2 of the pins.

Reply:
https://helsinkicity.slack.com/archives/CMKU6RAUX/p1584952087065700

Reply:
Thanks! It was just hard for me to see the difference from what it used to be before